ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -0,0 +1,155 @@
+    Should Be Equal    "${rc}"    "${expected_result}"    msg=${resource} does not exist in ${namespace}
+
+Is DSCI In Ready State
+    [Documentation]    Checks that DSCI Reconciled Succesfully
+    [Arguments]    ${dsci}    ${namespace}
+    ${rc}=    Run And Return Rc
+    ...    oc wait --timeout=3m --for jsonpath='{.status.conditions[].reason}'=ReconcileCompleted -n ${namespace} dsci ${dsci}
+    Should Be Equal As Integers    ${rc}     ${0}    msg=${dsci} not in Ready state
+
+Check ConfigMap Contains CA Bundle
+    [Documentation]    Checks that ConfigMap contains CA Bundle
+    [Arguments]    ${config_map}    ${ca_bundle_name}    ${namespace}
+    ${rc}=     Run And Return Rc
+    ...    oc get configmap ${config_map} -n ${namespace} -o yaml | grep ${ca_bundle_name}
+    Should Be Equal    "${rc}"    "0"    msg=${TRUSTED_CA_BUNDLE_CONFIGMAP} does not contain CA bundle ${ca_bundle_name}
+
+Is CA Bundle Value Present
+    [Documentation]    Check if the ConfigtMap contains Custom CA Bundle value
+    [Arguments]    ${config_map}    ${custom_ca_bundle_value}    ${namespace}        ${expected_result}
+    ${rc}=     Run And Return Rc
+    ...    oc get configmap ${config_map} -n ${namespace} -o yaml | grep ${custom_ca_bundle_value}
+    Should Be Equal As Integers    ${rc}    ${expected_result}
+
+Create Namespace In Openshift
+    [Documentation]    Create a new namespace if it does not already exist
+    [Arguments]    ${namespace}
+    ${rc}=     Run And Return Rc    oc get project ${namespace}
+    IF    ${rc} != 0
+        ${rc}=     Run And Return Rc    oc new-project ${namespace}
+        Should Be Equal    "${rc}"    "0"    msg=Failed to create namespace ${namespace}
+    END
+
+Delete Namespace From Openshift
+    [Documentation]    Delete namespace from opneshift
+    [Arguments]    ${namespace}
+    ${rc}=     Run And Return Rc    oc delete project ${namespace}
+    Should Be Equal    "${rc}"    "0"    msg=Failed to delete namespace ${namespace}
+
+Set Custom CA Bundle Value In DSCI
+    [Documentation]    Set Custom CA Bundle Value in DSCI
+    [Arguments]    ${custom_ca_bundle_value}    ${namespace}
+    ${rc}=     Run And Return Rc
+    ...    oc patch DSCInitialization/default-dsci -n ${namespace} -p '{"spec":{"trustedCABundle":{"customCABundle":"${custom_ca_bundle_value}"}}}' --type merge
+    Should Be Equal    "${rc}"    "0"    msg=Failed to set DSCI Custom CA Bundle value to ${custom_ca_bundle_value}
+
+Set Custom CA Bundle Value On ConfigMap
+    [Documentation]    Set Custom CA Bundle Value in ConfigMap
+    [Arguments]    ${config_map}    ${custom_ca_bundle_value}    ${namespace}
+    ${rc}=     Run And Return Rc
+    ...    oc patch ConfigMap/${config_map} -n ${namespace} -p '{"data":{"odh-ca-bundle.crt":"${custom_ca_bundle_value}"}}' --type merge
+    Should Be Equal    "${rc}"    "0"    msg=Failed to set ${config_map} value ${custom_ca_bundle_value} ${namespace}
```

Comment:
suggestion for here and other places where the same logic is applied: use `Run And Return Rc And Output` instead. In this way, in case of failure we can see the original error message in the logs. 

You could also `Should Be Equal "${rc}" "0" msg=${output}`

Reply:
Fixed.